aws/cloudformation/iam.yml.erb

```
@@ -225,15 +273,29 @@ Resources:
             Condition:
               StringEquals:
                 SAML:aud: https://signin.aws.amazon.com/saml
-      ManagedPolicyArns:
-        - !Ref DataAnalystPolicy
-        - !Ref DataAnalystDmsReadOnlyPolicy
-        - !Ref DataAnalystRedshiftPolicy
-        - arn:aws:iam::aws:policy/AmazonRedshiftQueryEditor
-        - arn:aws:iam::aws:policy/AmazonRedshiftQueryEditorV2FullAccess
+          <%if config.GoogleIdsSSMParameter -%>
```

I believe we'll actually need to use a hash reference with a symbol argument:

```suggestion
          <%if config[:GoogleIdsSSMParameter] -%>
```

For this and all other `config` references, including the ones that I implemented with strings in my original suggestion.